### PR TITLE
Added an example soft and hard eviction configuration

### PR DESCRIPTION
**Pull request by ahardin-rh**
Adds on to https://github.com/openshift/openshift-docs/pull/3384 addressing https://bugzilla.redhat.com/show_bug.cgi?id=1400887

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@derekwaynecarr PTAL

**Comment by ahardin-rh** [comment]
@derekwaynecarr Can you please tech review this? Further comments are in the BZ:
https://bugzilla.redhat.com/show_bug.cgi?id=1400887#c19

Thank you!

File: admin_guide/out_of_resource_handling.adoc [hunk 1]
```diff
@@ -168,6 +168,27 @@ To configure hard eviction thresholds, the following flag is supported:
...
+==== Soft and Hard Eviction Example Configuration
+
+.Soft and Hard Eviction Example Configuration on a 32Gi Node
+----
+eviction-hard:
+- "memory.available<100Mi"
+eviction-soft:
+- "memory.available<1Gi"
+eviction-soft-grace-period:
+- "memory.available=60s"
```
Comment (sferich888) [inline admin_guide/out_of_resource_handling.adoc]: We probably don't want to be recommending changes to this setting. 
We likely want to be recommending that the default here is used (and explaining why).

**Comment by ahardin-rh** [comment]
@derekwaynecarr bump

**Comment by ahardin-rh** [comment]
@derekwaynecarr Please review

**Comment by derekwaynecarr** [comment]
@ahardin-rh -- this is not what we want.  i have a card that captures what we want in spirit here: https://trello.com/c/MKBBB6JQ/709-1-improve-documentation-for-out-of-resource-handling  - i will try and get some doc together soon.

**Pull request closed without merging**